fabs/src/main/java/com/decathlon/vitamin/compose/fabs/VitaminFabs.kt

```diff
@@ -0,0 +1,136 @@
+package com.decathlon.vitamin.compose.fabs
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.FloatingActionButton
+import androidx.compose.material.FloatingActionButtonDefaults
+import androidx.compose.material.Icon
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material.ripple.LocalRippleTheme
+import androidx.compose.material.ripple.RippleTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminFabs {
+
+    /**
+     * The FAB must be used for primary, positive actions
+     * @param icon The icon to be displayed inside the FAB
+     * @param contentDescription The icon content description for accessibility
+     * @param modifier The [Modifier] to be applied to the component
+     * @param enabled True if you can click on the FAB, otherwise false
+     * @param colors The colors of the background and the content in enabled and disabled
+     * @param sizes The sizes for the icon and the component itself
+     * @param ripple The ripple effect applied on the component
+     * @param onClick The callback to be called when the user click on the FAB
+     */
+    @Composable
+    fun Primary(
+        icon: Painter,
+        contentDescription: String?,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: FabsColors = VitaminFabsColors.primary(),
+        sizes: FabSizes = VitaminFabSizes.default(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand,
```

Comment:
After design review, we prefer not overriding ripple effect because Material one is more accessible.
You can remove CompositionLocalProvider

Don't forget to update documentation & readme

Reply:
done